packages/server/src/profile/profile.controller.ts

```diff
@@ -176,4 +178,27 @@ export class ProfileController {
       },
     );
   }
+
+  @Delete('/delete_profile_picture')
+  async deleteProfilePicture(@User() user: UserModel): Promise<void> {
+    if (user.photoURL) {
+      fs.unlink(
+        process.env.UPLOAD_LOCATION + '/' + user.photoURL,
```

Comment:
dont' forget if you're testing locally to create an `uploads` folder in `package/server`